halo/_utils.py

```diff
@@ -4,7 +4,11 @@
 import codecs
 import platform
 import six
-from backports.shutil_get_terminal_size import get_terminal_size
+from sys import version_info
+if version_info.major == 2 or (version_info.major == 3 and version_info.minor < 3):
+    from backports.shutil_get_terminal_size import get_terminal_size
```

Comment:
We can add a version check for installing this dependency in `requirements.txt` itself:

```text
backports.shutil_get_terminal_size==1.0.0;python_version < '3.3'
```